addons/website/static/src/components/navbar/navbar.js

```diff
@@ -51,15 +64,21 @@ patch(NavBar.prototype, 'website_navbar', {
                         icon: 'fa-clone',
                         classes: 'btn-link ml-auto',
                         click: function () {
-                            parent.dialogService.add(DuplicatePageDialog, { pageId, onClose: this.close.bind(this) });
+                            parent.dialogService.add(DuplicatePageDialog, {
+                                pageId: parent.getPageId(),
+                                onClose: this.close.bind(this),
+                            });
```

Comment:
Maybe we could have a commit to modify the PageProperties dialog, that would be squashed with the previous commit introducing it. Ideally the Navbar patch would only include the code that displays and instantiates the extra website dropdowns, but not their business logic directly, so we could have some kind of PagePropertiesDialogWrapper component defined in dialogs/pages_properties, that has all the options, and we import it here and the Navbar patch only has the openWidget method that starts it.
It could be started the same way as the AceEditorComponent. We can discuss it tomorrow if you want